content/javascript/concepts/arrays/terms/push/push.md

```diff
@@ -12,42 +12,40 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Adds one or more elements to end of array and returns new length.
+The **`.push()`** method adds one or more elements to end of array and returns new length.
 
 ## Syntax
 
-The `.push()` method is not to be confused with returning an entirely new array with the passed object.
-
 ```js
-array.push(item);
+array.push(item1, item2, ...itemN);
 ```
 
-Multiple elements can be passed to end of array.
+A comma-separated list of elements can be passed to end of the `array`. The `.push()` method is not to be confused with returning an entirely new array with the passed object.
```

Comment:
```suggestion
A comma-separated list of elements can be passed to the end of the `array`. Variable can also be passed as an argument in the `push()` function. The `.push()` method is not to be confused with returning an entirely new array with the passed object.
```

Reply:
Interesting. I went with a similar change in 65801c9987f2adf2ba75ac072448cccf6b45d40a, instead, since you can technically do the same things with functions.

Reply:
> @Dusch4593 Very detailed commit messages

The truth is the truth lol